File: game-server/src/main/java/work/fking/pangya/game/packet/handler/LoginBonusClaimPacketHandler.java
package work.fking.pangya.game.packet.handler;

import io.netty.buffer.ByteBuf;
import work.fking.pangya.game.net.ClientGamePacketHandler;
import work.fking.pangya.game.GameServer;
import work.fking.pangya.game.player.Player;
import work.fking.pangya.game.packet.outbound.LoginBonusClaimReplyPacket;

public class LoginBonusClaimPacketHandler implements ClientGamePacketHandler {

    @Override
    public void handle(GameServer server, Player player, ByteBuf packet) {
        player.channel().writeAndFlush(new LoginBonusClaimReplyPacket());
    }
}

File: game-server/src/main/kotlin/work/fking/pangya/game/packet/handler/LoginBonusClaimPacketHandler.kt
package work.fking.pangya.game.packet.handler

import io.netty.buffer.ByteBuf
import work.fking.pangya.game.GameServer
import work.fking.pangya.game.net.ClientPacketHandler
import work.fking.pangya.game.packet.outbound.LoginBonusClaimReplyPacket
import work.fking.pangya.game.player.Player

class LoginBonusClaimPacketHandler : ClientPacketHandler {
    override fun handle(server: GameServer, player: Player, packet: ByteBuf) {
        player.writeAndFlush(LoginBonusClaimReplyPacket())
    }
}
